frontend/src/wizards/Governance/Policy/PolicyWizard.tsx

```diff
@@ -392,6 +403,77 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
           />
         </WizHidden>
 
+        {/* OperatorPolicy */}
+        <WizHidden hidden={(template: any) => template?.objectDefinition?.kind !== 'OperatorPolicy'}>
+          <div>
+            <Title headingLevel="h6">{t('Operator Policy')}</Title>
+            <Text component="small">{t('A operator policy creates operators on managed clusters.')}</Text>
+          </div>
+
+          <WizTextInput
+            path="objectDefinition.metadata.name"
+            label={t('Name')}
+            required
+            helperText={t('Name needs to be unique to the namespace on each of the managed clusters.')}
+            validation={validateKubernetesResourceName}
+          />
+          <Form>
+            <FormFieldGroupExpandable
+              isExpanded
+              header={
+                <FormFieldGroupHeader titleText={{ text: 'Operator Subscription', id: 'form-field-group-sub' }} />
+              }
+            >
+              <WizTextInput
+                path="objectDefinition.spec.subscription.name"
+                label={t('Name')}
+                labelHelp={t(
+                  'This is the package name of the Operator to install, which may be different than the Display Name used in the catalog.'
+                )}
+                required
+                helperText={t('Subscription name should be correct one')}
```

Comment:
Where does this text appear?

Reply:
Honestly I couldn't find on Console but Every forms has this one

Reply:
Let's remove ths